sdk/forms/src/main/java/com/klaviyo/forms/InAppFormsConfig.kt

```diff
@@ -13,7 +13,7 @@ import kotlin.time.Duration.Companion.seconds
  *  To disable session timeout altogether, use [Duration.INFINITE]
  */
 data class InAppFormsConfig(
-    private val sessionTimeoutDuration: Duration = DEFAULT_SESSION_TIMEOUT
+    private val sessionTimeoutDuration: Duration
```

Comment:
Kotlin duration doesn't translate nicely to Java unfortunately. As a result, you couldn't even call this constructor from Java, which cascaded up to not even being able to call `registerForInAppForms`. These secondary constructors make it possible to use `InAppFormsConfig` from Java without introducing any breaking changes to Kotlin code.